src/components/Dialogs/FileBrowser/ImageSave/ImageSaveComponent.tsx

```diff
@@ -174,23 +161,36 @@ export class ImageSaveComponent extends React.Component {
                         {numChannels > 1 && (
                             <React.Fragment>
                                 <div className="coordinate-select">
-                                    <FormGroup label={"Range unit"} inline={true}>
-                                        <HTMLSelect
-                                            value={activeFrame && (activeFrame.spectralCoordinate || "")}
-                                            options={spectralCoordinateOptions}
-                                            onChange={(event: React.FormEvent<HTMLSelectElement>) => this.updateSpectralCoordinate(event.currentTarget.value as string)}
-                                        />
-                                        <HTMLSelect
-                                            value={activeFrame && (activeFrame.spectralSystem || "")}
-                                            options={spectralSystemOptions}
-                                            onChange={(event: React.FormEvent<HTMLSelectElement>) => this.updateSpectralSystem(event.currentTarget.value as SpectralSystem)}
+                                    {activeFrame?.spectralSystemsSupported.length > 0 && (
+                                        <SpectralSettingsComponent
```

Comment:
Good change for reusing `SpectralSettingsComponent` for the UI. 👍

I wonder if we can modify related states in `FrameStore` so that `SpectralSettingsComponent` in general support native <-> channel conversion. This will allow conversion in z profiler, moment generator, PV generator, ... .

![image](https://github.com/CARTAvis/carta-frontend/assets/43841102/344497fb-90a4-4e85-b6cc-9e8b663769f4)